app/code/Magento/Sales/view/adminhtml/layout/sales_order_view.xml

```diff
@@ -64,7 +64,11 @@
 
                     <container name="payment_additional_info" htmlTag="div" htmlClass="order-payment-additional" />
                     <block class="Magento\Sales\Block\Adminhtml\Order\Payment" name="order_payment"/>
-                    <block class="Magento\Sales\Block\Adminhtml\Order\View\History" name="order_history" template="Magento_Sales::order/view/history.phtml"/>
+                    <block class="Magento\Sales\Block\Adminhtml\Order\View\History" name="order_history" template="Magento_Sales::order/view/history.phtml">
+                        <arguments>
+                            <argument name="viewModel" xsi:type="object">Magento\Sales\ViewModel\OrderHistoryComment</argument>
```

Comment:
What about changing the name of view model argument. Main reason is that Magento `Template` class has doc block with the next part:
```
 * <block name="my.block" class="Magento\Framework\View\Element\Template" template="My_Module::template.phtml" >
 *      <arguments>
 *          <argument name="viewModel" xsi:type="object">My\Module\ViewModel\Custom</argument>
 *      </arguments>
 * </block>
 ```
In most cases custom developers use `viewModel` as argument and it's possible that some custom module break sales order view page by defining another same name `viewModel`

Reply:
@Den4ik ok, I'll rename argument.